lib/pbench/test/unit/server/test_indexer.py

```diff
@@ -22,12 +22,32 @@ def test_not_found():
 
     @staticmethod
     def test_fallbacks():
-        keyvals = {"name": "mybm"}
+        templ = "%benchmark_name%_UID"
+        res = ResultData.expand_uid_template(templ, dict(foo=1, bar=2))
+        assert res == templ
+
+        keyvals = {"benchmark_name": "mybm"}
         templ = "%benchmark_name%_UID"
         res = ResultData.expand_uid_template(templ, keyvals)
         assert res == "mybm_UID"
+
+        keyvals = {"name": "myname"}
+
```

Comment:
I found this blank line vaguely unsettling, until I realized you were using the same `keyvals` in all of the following tests. Perhaps a comment?